### PR TITLE
Issue #6749: aligned javadoc/xdoc for NoWhitespaceBefore

### PR DESCRIPTION
**Pull request by pbludov**
Issue #6749 (part of #5750)

No changes in the code.

### CODE REVIEW

File: src/xdocs/config_whitespace.xml [hunk 1]
```diff
@@ -1244,7 +1244,7 @@ public void foo(final char @NotNull [] param) {} // No violation
           <tr>
             <td>allowLineBreaks</td>
             <td>
-              whether whitespace is allowed if the token is at a linebreak
+              Control whether whitespace is flagged at linebreaks.
```
Comment (rnveach) [inline src/xdocs/config_whitespace.xml]: > allowed if the token is at a linebreak
> flagged at linebreaks

I am on the fence about which wording to use.
Property is named `allow` so the word `flagged` seems like the negative form of the property when reading.
Its like saying "disallowed at linebreaks` when we name the property `allow`.

Comment (romani) [reply src/xdocs/config_whitespace.xml]: agree, existing description is more clear as for me.

Comment (rnveach) [reply src/xdocs/config_whitespace.xml]: @pbludov please use the old description

Comment (pbludov) [reply src/xdocs/config_whitespace.xml]: done

**Pull request merged**